src/_guides/language/language-tour_migrated.md

```diff
@@ -363,6 +363,51 @@ assert(lineCount == null);
   _condition_ is false. For details, see [Assert](#assert).
 {{site.alert.end}}
 
+
+### Late variables
+
+Dart 2.12 added the `late` modifier, which has two use cases:
+
+* To tell Dart that a non-nullable variable is set before it's used.
```

Comment:
"Is" might lead a reader astray, when it's really something more like "should be". If they forget to actually set it, the language won't stop them and will instead blow up at runtime.

Maybe something like: "To allow you to declare a non-nullable variable that is initialized after its declaration."

Reply:
How about:

* Declaring a non-nullable variable that's initialized after its declaration.
* Lazily initializing a variable.